tslint.json

```diff
@@ -50,8 +47,9 @@
     "no-switch-case-fall-through": true,
     "no-trailing-whitespace": true,
     "no-unused-expression": true,
+    "no-unused-variable": true,
     "no-use-before-declare": true,
-    "no-var-keyword": true,
```

Comment:
why?

Reply:
we use let and const instead

Reply:
Exactly. And `"no-var-keyword": true,` forces using `let` and `const`. By removing it we allow `var` :)

Reply:
sorry my mistake,  :upside_down_face: